### PR TITLE
Find Lat/Lng of EarthInteraction points.  

### PR DESCRIPTION
**Pull request by knowuh**
Still more to do on this honestly (like add the Lat/Lng to the location panel)

Actually the bulk of this work was just pulling out the InteractionController ...

### CODE REVIEW

File: Assets/Materials/Earth_new.mat [hunk 1]
```diff
@@ -68,7 +68,7 @@ Material:
     - _MainTex:
         m_Texture: {fileID: 2800000, guid: adbb30865c00243c09a85478a392e11b, type: 3}
         m_Scale: {x: 1, y: 1}
-        m_Offset: {x: 0, y: 0}
+        m_Offset: {x: 0.25, y: 0}
```
Comment (dstrawberrygirl) [inline Assets/Materials/Earth_new.mat]: We'll need to offset the gloss and specular to match this

File: Assets/Scripts/InteractionController.cs [hunk 2]
```diff
@@ -0,0 +1,125 @@
+﻿using UnityEngine;
+using System.Collections;
+
+
+public class InteractionController : MonoBehaviour
+{
+    public GameObject interactionIndicator;
+    public NetworkController _networkController;
```
Comment (dstrawberrygirl) [inline Assets/Scripts/InteractionController.cs]: This one should be private

File: Assets/Scripts/InteractionController.cs [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+            string interactionInfo = "local celestial interaction CO:" +
+                                     coName + ", " + coGroup + ", " + uniqueId;
+            CCLogger.Log(CCLogger.EVENT_ADD_INTERACTION, interactionInfo);
+        }
+    }
+
+    public void ShowEarthMarkerInteraction(Vector3 pos, Quaternion rot, Color playerColor, bool isLocal)
+    {
+        Vector2 latLng = Vector2.zero; // unset value
+        GameObject earth = GameObject.Find("Earth");
```
Comment (dstrawberrygirl) [inline Assets/Scripts/InteractionController.cs]: Minor, but a tag is probably better than a general find. I'd also recommend caching the reference so we don't have to look up every time. If null, you can go ahead and look it up again. Optimal approach is to do this once, then refresh the lookup OnSceneChange but it's a minor overhead so may not be worth the effort!

**Comment by knowuh** [comment]
Great feedback @dstrawberrygirl -- Latest two commits fix the code and scene respectively.

9019321 - InteractionController caching / tag lookup of earth.
00585fa - Add Earth tags where needed / adjusted new earth material offsets.

**Pull request merged**